### PR TITLE
[Notifier] [OvhCloud] Add `no_stop_clause` to DSN

### LINKED ISSUE #45932: [Notifier] [OvhCloud] Adding a no_stop_clause option
### Description

Hey, thanks for the great Notifier component !

It would be great to have some control over the OVH API 'noStopClause' option, which is currently set to false in the `OvhCloudTransport` class. This would allow us to send commercial SMS's with an alphanumerical sender name but no "stop clause" at the end of the message, by setting this option to true.

What do you think could be done about that ?

Thanks !

### PR DESCRIPTION
**Pull request by alamirault**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | Fix #45932
| License       | MIT

This PR allow to configure `no_stop_clause` (default false) for OvhCloud. Was always false before.

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php [hunk 1]
```diff
@@ -116,7 +127,7 @@ protected function doSend(MessageInterface $message): SentMessage
         if (200 !== $statusCode) {
             $error = $response->toArray(false);
 
-            throw new TransportException(sprintf('Unable to send the SMS: %s.', $error['message']), $response);
+            throw new TransportException(sprintf('Unable to send the SMS: "%s".', $error['message']), $response);
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php]: This line should be reverted

**Review by OskarStark — APPROVED** [review]
Looks good after my comment

File: src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php (hunk 1 above)
Comment (alamirault) [reply src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php]: Sure ? It's introduced by fabbot patch

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php]: Yes, it's a false positive in this case

File: src/Symfony/Component/Notifier/Bridge/OvhCloud/README.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ where:
  - `CONSUMER_KEY` is your OvhCloud consumer key
  - `SERVICE_NAME` is your OvhCloud service name
  - `SENDER` is your sender (optional)
+ - `NO_STOP_CLOSE` setting this parameter to "1" (default "0") allow removing "STOP clause" at the end of the message for non-commercial use (optional)
```
Comment (pbories) [inline src/Symfony/Component/Notifier/Bridge/OvhCloud/README.md]: NO_STOP_CLAUSE

**Review by OskarStark — APPROVED** [review]

**Review by welcoMattic — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alamirault.

**Pull request merged**